src/components/GarageAPI.ts

```diff
@@ -0,0 +1,123 @@
+export default class GarageAPI {
+  baseUrl: string;
+
+  path: {garage: string, winners: string, startStop: string};
+
+  constructor() {
+    this.baseUrl = 'http://localhost:3000';
+    this.path = {
+      garage: '/garage',
+      winners: '/winners',
+      startStop: '/engine',
+    };
+  }
+
+  async getCars() {
+    const response = await fetch(`${this.baseUrl}${this.path.garage}`);
+    const carsList = await response.json();
+    return carsList;
+  }
+
+  async getWinners() {
+    const response = await fetch(`${this.baseUrl}${this.path.winners}`);
+    const data = await response.json();
+    return data;
+  }
+
+  async getCarsCount() {
+    return (await this.getCars()).length;
+  }
+
+  async createCar(newCar: {name: string, color: string}) {
+    const response = await fetch(
+      `${this.baseUrl}${this.path.garage}`,
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(newCar),
+      },
+    );
+    const data = await response.json();
+    console.log(data, 'create');
```

Comment:
You should avoid of pushing debug logs. Better to add some rule to eslint to prevent this